src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -364,6 +365,15 @@ std::shared_ptr<ov::Model> FrontEnd::convert(const ov::frontend::InputModel::Ptr
 
     std::shared_ptr<ov::Model> f;
     translate_graph(model_tf, "TensorFlow_Frontend_IR", true, false, f);
+    apply_middle_transformations(f);
+
+    for (const auto& node : f->get_ordered_ops()) {
+        if (const auto& fw_node = ov::as_type_ptr<ov::op::util::FrameworkNode>(node)) {
+            FRONT_END_OP_CONVERSION_CHECK(false,
```

Comment:
Move this check after normalize pass, there is a comment asking for this.

Reply:
Done